dashboard/test/controllers/projects_controller_test.rb

```diff
@@ -180,10 +180,21 @@ class ProjectsControllerTest < ActionController::TestCase
     end
   end
 
-  test 'applab project level goes to edit if admin' do
+  test 'admins get redirected away' do
     sign_in create(:admin)
-    get :load, params: {key: 'applab'}
-    assert @response.headers['Location'].ends_with? '/edit'
+
+    get :index
+    assert_redirected_to '/'
+
+    %w(applab gamelab).each do |lab|
+      get :load, params: {key: lab}
+      assert_redirected_to '/'
+    end
+
+    %w(applab gamelab).each do |lab|
+      get :show, params: {key: lab, share: true, channel_id: 'fake_channel_id'}
+      assert_redirected_to '/'
+    end
```

Comment:
Not really (at least not enough to change it). The important thing is the user (admin) is not able to access the standalone project.